### PR TITLE
Evan prd 462 ranking signals quickstart notebook

### PR DESCRIPTION
**Pull request by ezeev**

### CODE REVIEW

File: notebooks/ranking_signals.ipynb [hunk 1]
```diff
@@ -0,0 +1,1005 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Objective Ranking Signals Quickstart\n",
+    "\n",
+    "Objective’s AI-native search ranks search results by a relevance score. Ranking Signals enable you to adjust the score, and therefore the final ranking of search results at query-time through a new query parameter.\n",
+    "\n",
```
Comment (fallguy) [inline notebooks/ranking_signals.ipynb:10]: Should we mention the name of the parameter? Like, "search results at query-time with the new `ranking_expr` parameter."

File: notebooks/ranking_signals.ipynb [hunk 2]
```diff
@@ -0,0 +1,1005 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Add Another Signal\n",
+    "\n",
+    "Our objects also have a `rating` attribute with values from 1 to 5. Let's extend our ranking signal so that we give a 2% boost to any objects having a rating equal to or greater than 4. To accomplis this, we'll follow the same pattern as our `Menswear` boost but we will add it to the existing expression by multiplying them together. This means that any object that is in the `Menswear` category with a rating of over 4 or more will receive a `4%` boost."
+   ]
```
Comment (fallguy) [inline notebooks/ranking_signals.ipynb:719]: we're going to denote `Menswear` using a code snippet we should also do the same for "rating" in the last sentence.

**Review by fallguy — COMMENTED** [review]
Added a few comments about the language. overall content looks great!

**Review by fallguy — APPROVED** [review]
Feel free to merge after addressing prior comments

File: notebooks/ranking_signals.ipynb [hunk 3]
```diff
@@ -0,0 +1,1005 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Objective Ranking Signals Quickstart\n",
+    "\n",
+    "Objective’s AI-native search ranks search results by a relevance score. Ranking Signals enable you to adjust the score, and therefore the final ranking of search results at query-time through a new query parameter.\n",
+    "\n",
+    "The most common use case for Ranking Signals is for query-time “boosting” or “de-boosting”. Boosting is commonly used for personalization based on user preferences or other contextual information that depends on external factors outside of the core, intrinsic relevancy of your search system.\n",
```
[the remark this answers is not in the document]
Comment (ezeev) [reply notebooks/ranking_signals.ipynb]: done

**Review by fallguy — APPROVED** [review]

**Review by fallguy — APPROVED** [review]
looks great!

**Review by JLHasson — APPROVED** [review]

**Pull request merged**